### ISSUE TITLE
Why Q=vote_candidate_features.unsqueeze(1).permute(1, 0, 2), K=V=torch.cat([key_features.permute(1, 0, 2), voxel_key_features.permute(1, 0, 2)], dim=0)

### ISSUE THREAD

**Issue by qifeng22** [Nightmare-n/PVT-SSD#12]
vote_features = self.pv_transformer(
            src=torch.cat([key_features.permute(1, 0, 2), voxel_key_features.permute(1, 0, 2)], dim=0),
            tgt=vote_candidate_features.unsqueeze(1).permute(1, 0, 2),
            pos_res=torch.cat([key_pos_emb.permute(1, 0, 2).unsqueeze(0), voxel_key_pos_emb.permute(1, 0, 2).unsqueeze(0)], dim=1)
        ).squeeze(0)
        
About transformer module, why you choose vote_candidate_features as Q, and torch.cat([key_features.permute(1, 0, 2), voxel_key_features.permute(1, 0, 2)], dim=0) as K ?
@Nightmare-n
